### DISCUSSION TITLE
The ability to set duration for particle animations to infinite

### DISCUSSION THREAD

**Discussion by DrakePork** [Zrips/CMI#2763] [Feature Requests]
**Description of feature/improvement request:**

The ability to set the duration: to infinite, for example duration: -1. By infinite I mean that the animation is on the player until the player either leaves the server or the server restart.

---
### Detailed explanation of what it should do, when and in what situation: ###
If you for example did /panimation circle;effect:heart;dur:-1, then the particle effect should stay on the player until the player leaves or the server restarts.

### Basic explanation as to why this feature is needed and/or useful to have: ###
This would be useful if you for example wanted to make like a trail of for example hearts after a player that didnt vanish after X time.

**Comment by Va1a** [comment]
i am pretty sure if you don't use the "dur" value at all, the animation remains until it is manually removed. could be wrong though.

**Comment by DrakePork** [comment]
> 
> 
> i am pretty sure if you don't use the "dur" value at all, the animation remains until it is manually removed. could be wrong though.

If no duration is specified the command just uses default time which is 5 seconds
